src/xrEngine/mailSlot.cpp

```diff
@@ -68,7 +68,7 @@ void msWrite(char* name, char* dest, char* msg)
 #ifndef LINUX // FIXME!!!
     HANDLE hFile;
     DWORD cbWritten;
-    BOOL fResult;
+    bool fResult;
```

Comment:
Let's leave it as is.
```suggestion
    BOOL fResult;
```